examples/virtual_keyboard/rust/main.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+slint::include_modules!();
+
+#[cfg_attr(target_arch = "wasm32", wasm_bindgen(start))]
+pub fn main() {
+    let main_window = MainWindow::new().unwrap();
+
+    virtual_keyboard::init(&main_window);
+
+    let _ = main_window.show();
+    let _ = main_window.run();
```

Comment:
I think that we should either unwrap or use a main that returns the error, instead of silently ignoring it. I think the latter would complicate the #cfg'ery a little, so I prefer unwrap.